src/universal/components/QueryRenderer/QueryRenderer.js

```diff
@@ -353,15 +348,15 @@ function fetchQueryAndComputeStateFromProps(
     const operation = createOperationDescriptor(request, variables)
     const relayContext = getContext(genericEnvironment, operation.variables)
     const queryKey = getRequestCacheKey(request.params, props.variables)
-    ReactRelayQueryRenderer.renewTTL(queryKey)
-    if (props.subscriptions && environment.registerQuery) {
-      environment.registerQuery(
-        queryKey,
-        props.subscriptions,
-        props.subParams,
-        operation.variables,
-        queryFetcher
-      )
+    const cacheQuery = Boolean(
```

Comment:
Stupid Javascript question: is that `Boolean()` needed with those logical operators? Is it because `environment.registeryQuery` could be undefined, and the rest will be short circuited?

Reply:
it's not necessary, i just do it so typescript knows it's a boolean, otherwise it would be `undefined | number`